tests/integration/components/em-select-test.js

```diff
@@ -101,6 +105,48 @@ test('em-select can select an item', function(assert) {
 
 });
 
+test('em-select can select an item when not disabled', function(assert) {
+
+  this.set('fruitOptions', fruitOptions);
+  this.set('fruitSalad', fruitSalad);
+
+  this.render(hbs`{{#em-form as |form|}}{{form.select label="Fruits:" content=fruitOptions optionValuePath='id' optionLabelPath='name' prompt='None' property='favoriteFruit' model=fruitSalad}}{{/em-form}}`);
+
+  const element = this.$();
+  assert.equal(element.find('label:contains("Fruits:")').length, 1, 'label has for property');
+
+  const select = element.find('select')[0];
+  assert.equal(select.options.length, 6, 'select has correct amount of options');
+
+  Ember.run(() => {
+    this.$(select).val('5');
+    this.$(select).trigger('change');
+  });
+  assert.equal(fruitSalad.get('favoriteFruit'), 5, 'model favorite fruit is the selection');
+
+});
+
+test('em-select can\'t select an item when disabled', function(assert) {
+
+  this.set('fruitOptions', fruitOptions);
+  this.set('fruitSalad', fruitSalad);
+
+  this.render(hbs`{{#em-form as |form|}}{{form.select label="Fruits:" content=fruitOptions optionValuePath='id' optionLabelPath='name' optionValuePath='disabled' prompt='None' property='favoriteFruit' model=fruitSalad}}{{/em-form}}`);
+
+  const element = this.$();
+  assert.equal(element.find('label:contains("Fruits:")').length, 1, 'label has for property');
+
+  const select = element.find('select')[0];
+  assert.equal(select.options.length, 6, 'select has correct amount of options');
+
+  Ember.run(() => {
+    this.$(select).val('5');
+    this.$(select).trigger('change');
+  });
+  assert.equal(fruitSalad.get('favoriteFruit'), null, 'model favorite fruit is the selection');
```

Comment:
I want this to still be "none" because it should not change the state if the option beeing selected is disabled but javascript can select disabled options 😢

Reply:
Now at least it doesn't update the value if it is disabled.